crates/forge/src/main.rs

```diff
@@ -296,6 +269,7 @@ fn main() {
 #[cfg(test)]
 mod tests {
     use super::*;
+    use camino::Utf8PathBuf;
```

Comment:
Is this needed?

Reply:
yup, it was moved from the section above since Utf8PathBuf is now used only in the test